aqsconverters/aq.py

```diff
@@ -178,7 +179,7 @@ def produce_annotation(self, aq_query_type, *args, **kwargs):
         run.aq_kwargs = kwargs
 
         log_renku_aqs(
-            RunSchema().dumps(run), str(self.__hash__()), 
+            RunSchema().dumps(run), str(run_id),
```

Comment:
Ok, makes sense. 

By the way, I am now a bit confused that there is a new run for each `produce_annotation` i.e. for each separate `aqs_query_object` et al. But that's a different point. Maybe a TODO?

Presumably, run should be once for `renku run`.

Reply:
each execution of `produce_annotation` instantiate a new `Run` object, is that what you men ?

Reply:
as of now - yes. I am not sure this is what renku `Run` is meant to represent. If anything, this is more like `AstroQuery` - a request. There can be many of those per `Run`. They should be linked to `Run`. Maybe later.

Reply:
I will add a TODO to keep it in mind